### DISCUSSION TITLE
월 달력의 이름을 어떻게 정할까요?

### DISCUSSION THREAD

**Discussion by simeunseok** [boostcampwm-2021/android01-CalendaRun#194] [Q&A]
이전에 말했다시피 한 달력에 체크 포인트가 있을 경우 달력이 네 단위로 쪼개지지만 체크 포인트 이름은 셋이라서 한 자리가 비게됩니다.

지금은 임시로 마지막 달력에는 캘린더의 이름을  사용하고 있는데, 이름을 어떻게 변경하는게 좋을까요?

1번 지금처럼 마지막 달력의 이름은 캘린더 이름으로 사용한다.

2번 마지막 달의 이름을 추가로 입력 받는다.

3번 "마무리"와 같은 이름을 디폴트로 사용하게 한다.

또 다른 의견 있으면 달아주셨으면 좋겠습니다!

**Comment by Cha-Ji** [comment]
1번도 큰 문제가 없지만, 최종적으로는 2번이 좋을 것 같습니다.
수능을 예시로 들면 각종 모의고사가 체크포인트고, 마지막 달은 수능이 될텐데 이 때는 캘린더 이름을 써도 될 것 같구요.
회사를 예시로 들면 마지막 달이 퇴사가 될 수 있죠. 이런 예시에서는 이름이 다를 수 있을 것 같아요.
